test/Domain/LeanCode.DomainModels.MassTransitRelay.Tests/StoreAndPublishEventsTests.cs

```diff
@@ -89,9 +89,12 @@ private Task<List<RaisedEvent>> GetRaisedEvents()
                 .ToListAsync();
         }
 
-        public void Dispose() => dbContext.Dispose();
+        public void Dispose()
+        {
+            dbContext.Dispose();
+        }
```

Comment:
This change seems to be quite unnecessary.

Reply:
Yep, I did add the `GC.SupressFinalize` and then removed it, forgetting about the layout change. Will change back.